### PR TITLE
Add CNV platform detection

### PR DESCRIPTION
**Pull request by dinhquochan**
Customers detect our products 👍 

Thanks.

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -1597,6 +1597,17 @@
 			"icon": "Cloudera.png",
 			"website": "http://www.cloudera.com"
 		},
+		"CNV Platform": {
+			"cats": [
+				"1"
+			],
+			"headers": {
+				"Set-Cookie": "cnv_session"
```
Comment (jvoisin) [inline src/apps.json]: What about `"Set-Cookie": "^cnv_session"` instead?

**Comment by gadcam** [comment]
Are you a web agency or do you maintain a CMS?

File: src/apps.json (hunk 1 above)
Comment (dinhquochan) [reply src/apps.json]: Only `cnv_session` exactly 👍

**Comment by dinhquochan** [comment]
We are a web agency.

**Comment by AliasIO** [comment]
This has been merged manually, thank you.

**Pull request closed without merging**